test/bin/scenario.sh

```diff
@@ -197,22 +197,36 @@ launch_vm() {
     # FIXME: variable for vcpus?
     # FIXME: variable for memory?
     # FIXME: variable for ISO
-    if timeout "${VM_BOOT_TIMEOUT}" sudo virt-install \
+
+    local -r wait=$((VM_BOOT_TIMEOUT / 60))
+    local -r install_log=$(mktemp)
+    local -r install_log_dest="${WEB_SERVER_URL}/scenario-info/${SCENARIO}/vms/${vmname}/vm_install.log"
```

Comment:
WEB_SERVER_URL?

Reply:
Ah, right. I didn't check it, thought it's local fs...